packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -224,10 +224,14 @@ export class FrontendPluginImpl {
       bicepTemplateDir,
       FrontendPathInfo.ModuleProvisionFileName
     );
-    let provisionOrchestration = await fs.readFile(provisionFilePath, ConstantString.UTF8Encoding);
-    provisionOrchestration = compileHandlebarsTemplateString(provisionOrchestration, pluginCtx);
-    let provisionModules = await fs.readFile(moduleProvisionFilePath, ConstantString.UTF8Encoding);
-    provisionModules = compileHandlebarsTemplateString(provisionModules, pluginCtx);
+    const provisionOrchestration = await generateBicepFiles(
+      await fs.readFile(provisionFilePath, ConstantString.UTF8Encoding),
+      pluginCtx
+    );
+    const provisionModules = await generateBicepFiles(
+      await fs.readFile(moduleProvisionFilePath, ConstantString.UTF8Encoding),
```

Comment:
Confusion - the first argument of `generateBicepFiles` is filePath, but pass file content here?

Reply:
ok, thanks!